METRICS.md

```diff
@@ -119,34 +119,34 @@ If we have a list of all changes for every deployment, it is simple to calculate
 
 ```sql
 SELECT
- d.deploy_id,
- TIMESTAMP_TRUNC(d.time_created, DAY) as day,
- ##### Time to Change
- TIMESTAMP_DIFF(d.time_created, c.time_created, MINUTE) time_to_change_minutes
+  d.deploy_id,
+  TIMESTAMP_TRUNC(d.time_created, DAY) as day,
+  #### Time to Change
+  TIMESTAMP_DIFF(d.time_created, c.time_created, MINUTE) time_to_change_minutes
 FROM four_keys.deployments d, d.changes
 LEFT JOIN four_keys.changes c ON changes = c.change_id;
 ```
 
 From this base, we want to extract the daily median lead time to change. 
 
 ```sql
```

Comment:
Thank you!  I think it might be good to include both versions of this.  The first one clearly builds off of the core of the example above, and your refactor is more efficient and works better in the dashboard.  So it would be good to say "From this base, we extract this." . . . "Here is how we write it more efficiently for the dashboard."  

What do you think?

Reply:
Thanks for your comment!

I agree with separating the base query and dashboard query in METRICS.md, or only the base query. In my version, the dashboard divides by 60 (`IFNULL(ANY_VALUE(med_time_to_change)/60, 0) AS median_time_to_change, # Hours`), so I thought it would be easier to understand if this was also removed from the base query.

One thing I noticed is that the results are different for `GROUP BY day, time_to_change_minutes` and `GROUP BY day`. If we include time_to_change_minutes in GROUP BY as in the current version, we will end up with an aggregation of records with the same time_to_change_minutes on the same day. Since that scenario is quite possible, I thought it would be better to run the query without including time_to_change_minutes in GROUP BY even in the base query.

Reply:
Thanks for your comment!

One thing I am wondering is that `GROUP BY day, time_to_change_minutes` and `GROUP BY day` give different results. If we include time_to_change_minutes in GROUP BY as in the current version, it will aggregate records with the same time_to_change_minutes on the same day. 

Since such a scenario is quite possible, I thought it would be better to execute the query without including time_to_change_minutes in GROUP BY even in the query to extract daily median lead time. What do you think?

Reply:
In Commit https://github.com/GoogleCloudPlatform/fourkeys/pull/232/commits/5202e6a20b628459d664e426f428b67cc29d2d41, included both the query builded from the base query and the query created efficiently for the dashboard. and also slightly modified the query builded from the base query so that we get the same results for both queries.